pages/en/zkapps/how-to-write-a-zkapp.mdx

```diff
@@ -384,6 +384,104 @@ Please see the <a href="https://jestjs.io/docs/getting-started">Jest docs</a> fo
 
 We will be adding blockchain-specific testing functionality in the future. Keep an eye on this section for updates.
 
+## Deployment
+
+### Scaffolding an example zkApp to deploy
+
+To create an example zkApp, run the following command:
+
+```sh
+$ zk project myproj
+```
+
+This command will fetch the example project template and install all dependencies you need.
+
+### Create a new zkApp config
+
+We will use the Berkeley QA network as the network to deploy too. Firstly, you will need to create a config file that the zkApp CLI needs. To get started, change directories into the project and run the following command:
```

Comment:
Sorry, ended up with a number of nuanced changes, easier to communicate this way
___

> 
> Before deploying, we must first define which network that we will deploy to and give it a name. We do this by creating a network alias within your project's `config.json`.
> 
> First, change into the directory containing your project and then run the following command:
> 
> ```sh
> zk config
> ```
> 
> It will ask you to specify a name (can be anything) and URL to deploy to. The URL is the Mina GraphQL API that will receive your deploy transaction and broadcast it to the Mina network. Note that this URL is important because it also determines which network you're be deploying to (e.g. Testnet, Mainnet, etc)
> 
> For Berkeley QANet, let's use the following values:
> 
> > **Network Name**: `berkeley`
> > **URL**: `https://proxy.berkeley.minaexplorer.com/graphql`
> > **Fee**: `0.1`
> 
> <Alert kind="info">
> 
> If your project contains multiple smart contracts (e.g. `Foo` and `Bar`) that you intend to deploy to the same network, we recommend following a naming pattern such as `berkeley-foo` and `berkeley-bar` for your network alias names.
> 
> </Alert>
> 
> You will see the following output:

(Github's preview doesn't show it, but the 2nd last to paragraph is wrapped in an info box alert.